backend/lib/azimutt/tracking.ex

```diff
@@ -44,6 +44,36 @@ defmodule Azimutt.Tracking do
     |> Result.from_nillable()
   end
 
+  def recent_organization_events(%Organization{} = organization) do
+    allowed_events = [
+      "editor_layout_created",
+      "editor_layout_deleted",
+      "editor_memo_created",
+      "editor_memo_deleted",
+      "editor_memo_updated",
+      "editor_notes_created",
+      "editor_notes_deleted",
+      "editor_notes_updated",
+      "editor_source_added",
+      "editor_source_deleted",
+      "editor_source_refreshed",
+      "project_created",
+      "project_deleted",
+      "project_loaded",
```

Comment:
`project_loaded` is the only one that is read, and we may have a lot (in your screen you have only one which is not this one), not sure it's useful but 🤷

Reply:
Gardon le pour le moment, j'ai envie de voir ce que ca donne chez Fundimmo notamment. On verra les feedbacks par rapport à ca.